src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/forwardinghandler.cpp

```diff
@@ -2212,10 +2213,14 @@ FORWARDING_HANDLER::SetStatusAndHeaders(
                     break;
                 }
                 __fallthrough;
-            case HttpHeaderConnection:
             case HttpHeaderDate:
                 continue;
-
+            case HttpHeaderConnection:
```

Comment:
Do you need to check that the value is "close"?

Reply:
Or change the name to forwardResponseConnectionHeader?

Reply:
yeah I'll change the name to forwardResponseConnectionHeader.